### PR TITLE
PMM-9148 added 5 tests to verify new QAN Copy URL Button works

### PR DESCRIPTION
**Pull request by ElenaFedichkina**
Cases:
https://jira.percona.com/secure/Tests.jspa#/testCase/PMM-T1138
https://jira.percona.com/secure/Tests.jspa#/testCase/PMM-T1140
https://jira.percona.com/secure/Tests.jspa#/testCase/PMM-T1141
https://jira.percona.com/secure/Tests.jspa#/testCase/PMM-T1142
https://jira.percona.com/secure/Tests.jspa#/testCase/PMM-T1143

server docker - perconalab/pmm-server-fb:PR-2296-eded043
client docker - perconalab/pmm-client-fb:PR-2296-eded043
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2296-eded043.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2296-eded043&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2296-eded043.tar.gz

https://pmm.cd.percona.com/job/pmm2-ui-test-nightly/1023/
image

### CODE REVIEW

File: tests/QAN/timerange_test.js [hunk 1]
```diff
@@ -82,3 +84,116 @@ Scenario(
...
+    I.click(qanOverview.buttons.copyButton);
+
+    const url = await I.grabTextFrom(qanOverview.elements.clipboardLink);
+
+    I.openNewTab();
+    I.amOnPage(url);
+    qanOverview.waitForOverviewLoaded();
+    // this check will need to be uncommented after tasks 9480 and 9481 are ready
+    // qanPagination.verifyActivePage(2);
+    I.waitForVisible(qanOverview.getSelectedRowLocator(2), 20);
```
Comment (puneet0191) [inline tests/QAN/timerange_test.js:174]: @ElenaFedichkina  could you also make sure the Detail section is also opened automatically when opened in new tab

File: tests/QAN/timerange_test.js [hunk 2]
```diff
@@ -82,3 +84,116 @@ Scenario(
...
+
+    I.wait(30);
+    I.refreshPage();
+    qanOverview.waitForOverviewLoaded();
+    I.click(qanOverview.buttons.copyButton);
+
+    const url2 = await I.grabTextFrom(qanOverview.elements.clipboardLink);
+    const toTimeFromUrl2 = qanOverview.getToTimeFromUrl(url2);
+
+    assert.ok(Math.abs(toTimeFromUrl - toTimeFromUrl2) < 60000, 'Difference between moment time and second copied time must be less then one minute');
```
Comment (puneet0191) [inline tests/QAN/timerange_test.js:112]: also, add a check about toTimeFromUrl2 should not be the same as toTimeFromUrl1, it must at least have a difference of 30 seconds.

**Review by puneet0191 — CHANGES_REQUESTED** [review]

File: tests/QAN/timerange_test.js (hunk 1 above)
Comment (ElenaFedichkina) [reply tests/QAN/timerange_test.js:174]: ok

File: tests/QAN/timerange_test.js (hunk 2 above)
Comment (ElenaFedichkina) [reply tests/QAN/timerange_test.js:112]: ok.thanks

**Review by puneet0191 — APPROVED** [review]

File: tests/QAN/pages/qanOverviewFragment.js [hunk 3]
```diff
@@ -221,4 +225,12 @@ module.exports = {
     I.fillField(this.fields.searchBy, value);
     I.pressKey('Enter');
   },
+
+  getToTimeFromUrl(url) {
+    return url.slice(url.indexOf('&to=') + 4, url.indexOf('&page_number='));
```
Comment (yurkovychv) [inline tests/QAN/pages/qanOverviewFragment.js:230]: please use [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL_API#queries) to parse URLs. This way is not reliable

**Review by yurkovychv — CHANGES_REQUESTED** [review]

File: tests/QAN/pages/qanOverviewFragment.js (hunk 3 above)
Comment (ElenaFedichkina) [reply tests/QAN/pages/qanOverviewFragment.js]: ok. thank you.
edited

**Review by puneet0191 — APPROVED** [review]

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**